book/accessibility.md

```diff
@@ -2707,12 +2711,9 @@ compare the behavior with a real browser.
 
 [os-integ]: https://pypi.org/project/accessible_output/
 
-*Focus method and events*. Add support for the JavaScript
-[`focus()`][focus-method] method
-and the corresponding [`focus`][focus-event] and
-[`blur`][blur-event] events on DOM elements. Make sure that `focus()`
-only has an effect on focusable elements.
+14-10 *The `zoom` CSS property*. Add support for the [`zoom`][zoom-property] SSS
```

Comment:
Replacement exercise. `zoom` recently acquired a spec (just in 2024!), so it's now appropriate to include.